airbyte-workers/src/main/java/io/airbyte/workers/WorkerUtils.java

```diff
@@ -95,6 +99,15 @@ public static WorkerDestinationConfig syncToWorkerDestinationConfig(final Standa
         .withState(sync.getState());
   }
 
+  public static Map<String, JsonNode> mapStreamNamesToSchemas(final StandardSyncInput syncInput) {
+    final String streamPrefix = syncInput.getPrefix();
+    return syncInput.getCatalog().getStreams().stream().collect(
+        Collectors.toMap(
+            k -> String.format("%s".trim() + k.getStream().getName().trim(), Objects.toString(k.getStream().getNamespace(), "")),
```

Comment:
Can we extract the strings in var and get rid of the format, I don't see what that brings.

```
k -> {
   String namespace = Objects.toString(k.getStream().getNamespace(), ""));
   String name = k.getStream().getName().trim();

   return namespace + name;
}
```